doc/tutorials/introduction/crosscompilation/crosscompile_with_multiarch.markdown

```diff
@@ -0,0 +1,342 @@
+Cross compilation with Ubuntu/Debian{#tutorial_crosscompile_with_multiarch}
+====================================
+
+@prev_tutorial{tutorial_arm_crosscompile_with_cmake}
+@next_tutorial{tutorial_building_tegra_cuda}
+
+[TOC]
+
+|    |    |
+| -: | :- |
+| Original author | Kumataro |
+| Compatibility   | Ubuntu >=23.04 |
+|^                | OpenCV >=4.8.0 |
+
+@warning
+This tutorial can contain obsolete information.
+
+What is "MultiArch"
+-------------------
+Ubuntu and Debian supports MultiArch.
+It allows to install several foreign architecture libraries.
+
+(e.g. aarch64 and/or armv7 binaries are able to installed at x86-64 hosts.)
+
+MultiArch can be used to cross-compile OpenCV library too.
+
+@warning
+- Following these steps will make your Linux environment a little dirty.
+  If possible, it is better to use VMs.
```

Comment:
Yes, docker is one of selections. However there are another container technology. I felt that it would be better to include other container technologies. 

>  it is better to use VMs or Container(e.g. Docker).